pkg/workceptor/kubernetes.go

```diff
@@ -249,21 +249,26 @@ func (kw *kubeUnit) createPod(env map[string]string) error {
 
 	if err == ErrPodCompleted {
 		// Hao: shouldn't we also call kw.Cancel() in these cases?
-		if len(kw.pod.Status.ContainerStatuses) != 1 {
-			return fmt.Errorf("expected 1 container in pod but there were %d", len(kw.pod.Status.ContainerStatuses))
-		}
+		for _, cstat := range kw.pod.Status.ContainerStatuses {
```

Comment:
If I am reading the corev1.Pod code correctly, both the `Status` and `ContainerStatuses` fields are marked with `omitempty` 
What happens to this functions if `Status = nil` ?